### DISCUSSION TITLE
How to serialize model with camelCase properties?

### DISCUSSION THREAD

**Discussion by guoyunhe** [adonisjs/core#3973] [Help] [answered]
I want to have my model like this:

Model object:

```
dateOfBirth: '1991-09-09'
lastLogin: '2022-09-09'
```

Database table:

```
date_of_birth: '1991-09-09'
last_login: '2022-09-09'
```

HTTP request/response:

```
dateOfBirth: '1991-09-09'
lastLogin: '2022-09-09'
```

Is here any config option to make models serialized as camelCase globally?

**Comment by tomgobich** [comment, marked as answer]
You'll want to define your own naming strategy and apply it to your Models or the AdonisJS BaseModel
https://docs.adonisjs.com/reference/orm/naming-strategy#document
https://docs.adonisjs.com/reference/orm/base-model#static-namingstrategy

I also have a lesson on this on Adocasts, though we create our own `BaseModel` extending Adonis' in my lesson, either way works though.
https://www.youtube.com/watch?v=OAzvVHnsU8M
